### ISSUE TITLE
Please archive obsolete Catena-X repositories 'product-agents' and 'product-agents-edc'

### ISSUE THREAD

**Issue by drcgjung** [eclipse-tractusx/sig-infra#256]
**Is your support request related to a problem? Please describe.**  

Someone could be tempted to develop things against these outdated repos 
- https://github.com/catenax-ng/product-agents 
- https://github.com/catenax-ng/product-agents-edc

(they have been contributed to their new home eclipse-tractusx/knowledge-agents and eclipse-tractusx/knowledge-agents-edc and forked for collaboration into catenax-ng/tx-knowledge-agents and catenax-ng/tx-knowledge-agents-edc in the meantime).

**Describe the solution you'd like**  

the two outdated repos should be archived (such that the artifacts could still be downloaded in beta environments that have not yet been migrated to the 23.09 release)

Later (after the 23.12 release), the two repos can be deleted.

**Additional context**  

To be archived/deleted
- https://github.com/catenax-ng/product-agents 
- https://github.com/catenax-ng/product-agents-edc 

New official homes are
- https://github.com/eclipse-tractusx/knowledge-agents 
- https://github.com/eclipse-tractusx/knowledge-agents-edc

Collaboration/preparation forks are 
- https://github.com/catenax-ng/tx-knowledge-agents 
- https://github.com/catenax-ng/tx-knowledge-agents-edc

**Comment by hzierer** [comment]
Hi @drcgjung ,

shall we update the "about" sections accordingly with a link to the replacement/equivalent? If so, can you provide a list to the successors / best matches?

**Comment by drcgjung** [comment]
that's a nice hint.

I already did that (at least verbose and pointing to the tx-forks, not as a hyperlink to eclipse-tractusx ... if that would be possible, I would appreciate that)

Would it be possible to update the eclipse-tractusx repos About sections too?

knowledge-agents -> 'Reference Implementations of CX-0084'

knowledge-agents-edc -> 'EDC Extensions for CX-0084'

**Comment by hzierer** [comment]
yes, the tool of the EF also supports changing the description and hyperlinks are also no problem in there.

But I think we should avoid a simple notion of an internal name ("CX-0084") on a public repo - is this specification somewhere publicly available or maybe under another name?

**Comment by hzierer** [comment]
And just to be sure:
[catenax-ng/product-agents](https://github.com/catenax-ng/product-agents) "refers" to [catenax-ng/tx-knowledge-agents](https://github.com/catenax-ng/tx-knowledge-agents), which is a fork of [eclipse-tractusx/knowledge-agents](https://github.com/eclipse-tractusx/knowledge-agents). And after this tickets all 3 (6 with the -edc ones) should be archived (and ultimately deleted).
So the whole code is lost after the 23.12 release? Is that the intention?

**Comment by drcgjung** [comment]
> yes, the tool of the EF also supports changing the description and hyperlinks are also no problem in there.
> 
> But I think we should avoid a simple notion of an internal name ("CX-0084") on a public repo - is this specification somewhere publicly available or maybe under another name?

Fair enough, I think the standard will be published with the 23.09 Release under a hyperlink something like that:
https://catena-x.net/fileadmin/user_upload/Standard-Bibliothek/Archiv/Update_September_23_R_3.2/CX-0084-FederatedQueriesInDataSpaces-v1.0.0.pdf

Until the release wave is published, it could be found here

[CX-0084-FederatedQueriesInDataSpaces-v1.0.0.pdf](https://github.com/catenax-ng/product-catena-x-standardization/blob/main/standards/CX-0084-FederatedQueriesInDataSpaces/1.0.0/CX-0084-FederatedQueriesInDataSpaces-v1.0.0.pdf)

You could refer to it more complete as 'CX-0084 Federated Queries In Data Spaces'

**Comment by drcgjung** [comment]
> And just to be sure: [catenax-ng/product-agents](https://github.com/catenax-ng/product-agents) "refers" to [catenax-ng/tx-knowledge-agents](https://github.com/catenax-ng/tx-knowledge-agents), which is a fork of [eclipse-tractusx/knowledge-agents](https://github.com/eclipse-tractusx/knowledge-agents). And after this tickets all 3 (6 with the -edc ones) should be archived (and ultimately deleted). So the whole code is lost after the 23.12 release? Is that the intention?

God, no. The intention is just to deprecate/archive the catenax-ng/product-agents and catenax-ng/product-agents-edc repositories.

The eclipse-tractusx and their forks will stay forever ... at least will be maintained for 23.12

**Comment by hzierer** [comment]
ok, thank you for the clarification

**Comment by drcgjung** [comment]
I would have one last plea ... I know that this was not part of the original description.

The associated maven packages 

- https://github.com/catenax-ng/product-agents/packages/1877564
- https://github.com/catenax-ng/product-agents/packages/1877565
- https://github.com/catenax-ng/product-agents/packages/1877566
- https://github.com/catenax-ng/product-agents/packages/1877567

- https://github.com/catenax-ng/product-agents-edc/packages/1868802
- https://github.com/catenax-ng/product-agents-edc/packages/1868806
- https://github.com/catenax-ng/product-agents-edc/packages/1868805
- https://github.com/catenax-ng/product-agents-edc/packages/1868807
- https://github.com/catenax-ng/product-agents-edc/packages/1868808
- https://github.com/catenax-ng/product-agents-edc/packages/1868809
- https://github.com/catenax-ng/product-agents-edc/packages/1868810
- https://github.com/catenax-ng/product-agents-edc/packages/1865120
- https://github.com/catenax-ng/product-agents-edc/packages/1868800
- https://github.com/catenax-ng/product-agents-edc/packages/1868800
- https://github.com/catenax-ng/product-agents-edc/packages/1868799
- https://github.com/catenax-ng/product-agents-edc/packages/1868801

would need to be deleted too (else we cannot publish them under the tx- repos, that is a "bug" in github)

Thx much!

**Comment by almadigabor** [comment]
Hey @drcgjung! Deleted all the requested packages. Please confirm.

**Comment by drcgjung** [comment]
@almadigabor you are the best!!!!
